setup.py

```diff
@@ -11,8 +11,9 @@
     install_requires=[
         "prettytable>=0.7.2",
         "pysha3>=1.0.2",
-        # "crytic-compile>=0.2.1",
+        #"crytic-compile>=0.2.1",
         "crytic-compile",
+        "web3",
```

Comment:
I think we should not make this a permanent dep. `get_storage_slot_and_val` is already using a try/catch approach, the only other place where we used it is `get_storage_data`

We can probably move the `try/catch` import at the `read_storage.py` top-level, and have the tool not loading at all if the dep is not installed (see the comment on read_storage.py)